features/step_definitions/assertion_steps.rb

```diff
@@ -38,3 +43,7 @@
 Then("I should be on password reset page") do
   expect(current_path).to eq new_user_password_path
 end
+
+Then("I should be on My Profile page") do
+  expect(current_path).to eq user_path(@user)
```

Comment:
Where do you set the `@user` variable?

Reply:
we have this step in the feature file line 17 "And I am logged in as "maran@test.com""
this sets the @user variable

Reply:
Because `current_user` can't be used in step definitions

Reply:
Okay. Cool. Thank you